docs/build.md

```diff
@@ -18,11 +18,28 @@ sudo apt-get -qq update \
 
 Required Erlang OTP version is `20.1`.
 
-A compatible OTP version can be installed from a package both on Ubuntu 16.04 and Ubuntu 18.04:
+#### Ubuntu 18.04
+
 ```bash
 sudo apt-get install erlang
 ```
 
+#### Ubuntu 16.04
+
+Ubuntu 16.04 ships with outdated erlang version. Version 20 can be installed from source:
+
+```bash
```

Comment:
I see in line with tested snippet https://github.com/aeternity/docker-builder/blob/58b939c933304330938251cd58e31f6be1978de1/Dockerfile#L10-L17